Update style.css

jeffikus opened the pull request:
#### Changes proposed in this Pull Request:

Adds auto-loading-homepage tag to stylesheet. Fixes homepage switching on WordPress.com

Themes:
- Blockbase
- Geologist
- Mayland Blocks
- Payton
- Quadrat
- Russell
- Seedlet Blocks
- Skatepark
- Videomaker
- Zoologist


#### Related issue(s):

None.

jeffikus committed:
Update style.css

Adds auto-loading-homepage to stylesheet. Fixes homepage switching on WordPress.com

Themes:
- Blockbase
- Geologist
- Mayland Blocks
- Payton
- Quadrat
- Russell
- Seedlet Blocks
- Skatepark
- Videomaker
- Zoologist

pbking commented on payton/style.css:
```diff
@@ -12,6 +12,6 @@ License: GNU General Public License v2 or later
 Template: blockbase
 Text Domain: payton
-Tags:
+Tags:auto-loading-homepage
```
The .zip building script is specifically looking for `, auto-loading-homepage` thus if it is the first (or only) item it will not be removed.  Need to either add others so it's not the last or update (and complicate) the script.

jeffikus replied:
Added editor-styles tag.

pbking commented on videomaker/style.css:
```diff
@@ -12,6 +12,6 @@ License: GNU General Public License v2 or later
 Template: blockbase
 Text Domain: videomaker
-Tags: 
+Tags: auto-loading-homepage
```
same problem as payton, needs `, auto-loading-homepage` to be removed.

jeffikus replied:
Added editor-styles tag.

jeffikus committed:
Adds at least 1 tag before auto-loading-homepage

Adds at least 1 tag before auto-loading-homepage to account for the dotorg script.

pbking reviewed (approved):
Does the job.
.org script will strip it out finely
🚢
